atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/expectations/ExpectationsStatistics.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.api.expectations;
+
+import com.google.common.collect.ImmutableMap;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface ExpectationsStatistics {
```

Comment:
We should add docs to the interface.
Because this kind of reads like it is a per transaction model, but so is `TransactionReadInfo`